images/win/Windows2022-Archive_Grouped.json

```diff
@@ -0,0 +1,1594 @@
+      },
+      {
+        "Pipx": "Pipx 1.1.0"
+      },
+      {
+        "Pip": "pip 22.3 (python 3.9)"
+      },
+      {
+        "Ruby": "RubyGems 3.2.33"
+      },
+      {
+        "Vcpkg": "Vcpkg (build from master \\<79fa9d85c>)"
+      },
+      {
+        "Yarn": "Yarn 1.22.19"
+      }
+    ]
+  },
+  {
+    "Project Management": [
+      {
+        "Env_VCPKG_INSTALLATION_ROOT": "VCPKG_INSTALLATION_ROOT|C:\\vcpkg"
+      },
+      {
+        "Env_VCPKG_ROOT": "VCPKG_ROOT|C:\\vcpkg"
+      }
+    ]
+  },
+  {
+    "Tools": [
+      {
+        "Ant": "Ant 1.10.12"
+      },
+      {
+        "Gradle": "Gradle 7.5"
+      },
+      {
+        "Maven": "Maven 3.8.6"
+      },
+      {
+        "Sbt": "sbt 1.7.2"
+      }
+    ]
+  },
+  {
+    "CLI Tools": [
+      {
+        "7zip": "7zip 22.01"
+      },
+      {
+        "Aria2": "aria2 1.36.0"
+      },
+      {
+        "AzCopy": "azcopy 10.16.1"
+      },
+      {
+        "Bazel": "Bazel 5.3.2"
+      },
+      {
+        "Bazelisk": "Bazelisk 1.13.2"
+      },
+      {
+        "Bicep": "Bicep 0.11.1"
+      },
+      {
+        "Cabal": "Cabal 3.8.1.0"
+      },
+      {
+        "CMake": "CMake 3.24.2"
+      },
+      {
+        "CodeQLBundle": "CodeQL Action Bundle 2.11.0"
+      },
+      {
+        "Docker": "Docker 20.10.19"
+      },
+      {
+        "DockerCompose": "Docker Compose v1 1.29.2"
+      },
+      {
+        "DockerComposeV2": "Docker Compose v2 2.11.2"
+      },
+      {
+        "DockerWincred": "Docker-wincred 0.7.0"
+      },
+      {
+        "GHC": "ghc 9.4.2"
+      },
+      {
+        "Git": "Git 2.37.3.windows.1"
+      },
+      {
+        "GitLFS": "Git LFS 3.2.0"
+      },
+      {
+        "InnoSetup": "InnoSetup 6.2.1"
+      },
+      {
+        "JQ": "jq 1.6"
+      },
+      {
+        "Kind": "Kind 0.16.0"
+      },
+      {
+        "Kubectl": "Kubectl 1.25.3"
+      },
+      {
+        "Mercurial": "Mercurial 5.0"
+      },
+      {
+        "MinGW": "Mingw-w64 11.2.0"
+      },
+      {
+        "Newman": "Newman 5.3.2"
+      },
+      {
+        "NSIS": "NSIS v3.08"
+      },
+      {
+        "OpenSSL": "OpenSSL 1.1.1"
+      },
+      {
+        "Packer": "Packer 1.8.2"
+      },
+      {
+        "Pulumi": "Pulumi v3.43.1"
+      },
+      {
+        "R": "R 4.2.1"
+      },
+      {
+        "ServiceFabricSDK": "Service Fabric SDK 9.0.1028.9590"
+      },
+      {
+        "Stack": "Stack 2.9.1"
+      },
+      {
+        "SVN": "Subversion (SVN) 1.14.2"
+      },
+      {
+        "VSWhere": "VSWhere 3.0.3"
+      },
+      {
+        "Swig": "Swig 4.0.2"
+      },
+      {
+        "WinApp": "WinAppDriver 1.2.2009.02003"
+      },
+      {
+        "Wix": "WiX Toolset v3.11.2.4516"
+      },
+      {
+        "Zstd": "zstd 1.5.2"
+      },
+      {
+        "YAMLLint": "yamllint 1.28.0"
+      },
+      {
+        "ImageMagick": "ImageMagick 7.1.0-50"
+      }
+    ]
+  },
+  {
+    "Rust Tools": [
```

Comment:
It looks like that sections are shifted.

Reply:
corrected